fast-setup.sh

```diff
@@ -88,7 +88,9 @@ then
 fi
 
 # update source.sh
-echo "export RISCV=$(pwd)/riscv${BITS}" > ./source.sh
+GCC_PATH=$(whereis riscv$BITS-unknown-linux-gnu-gcc)
```

Comment:
can you use `which` instead so that we can remove `cut` command below?
e.g., GCC_PATH=$(which riscv$BITS-unknown-linux-gnu-gcc)

Reply:
Fixed